Library/Formula/twister.rb

```diff
@@ -0,0 +1,56 @@
+require "formula"
+
+class Twister < Formula
+  homepage "http://twister.net.co"
+  url "https://github.com/miguelfreitas/twister-core/archive/v0.9.28.tar.gz"
+  sha1 "53b03636ae4d7539002fe9c3cf2dd58e9657ac1e"
+
+  head do
+    url "https://github.com/miguelfreitas/twister-core.git"
+  end
+
+  depends_on "boost" => :build
+  depends_on "miniupnpc" => :build
+  depends_on "openssl" => :build
+  depends_on "berkeley-db4" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  option "without-webui", "Build without web ui"
+
+  def install
+    if build.head?
+      ln_s cached_download/".git", ".git"
+    end
+
+    system "./autotool.sh"
+
+    args = %W[
+      --prefix=#{prefix}
+      --sysconfdir=#{etc}
+      --with-openssl=#{Formula["openssl"].opt_prefix}
+    ]
+
+    system "./configure", *args
+    system "make", "install"
+
+    if build.with? "webui"
+      html_dir = "#{ENV['HOME']}/Library/Application\ Support/Twister/html"
+
+      if File.exists? html_dir
+        File.rename html_dir, "#{ENV['HOME']}/Library/Application\ Support/Twister/html.bak-#{Time.now.strftime '%Y%m%d%H%M%S'}"
+      end
+
+      mkdir_p html_dir
+      system "git clone https://github.com/miguelfreitas/twister-html.git ${HOME}/Library/Application\\ Support/Twister/html"
```

Comment:
Could always stick a note in ` def caveats ` to say if you want to use the WebUI feature you should do ` cd blah && git clone blah ` I guess?